### DISCUSSION TITLE
Profile check-in

### DISCUSSION THREAD

**Discussion by otter-computer** [community/community#50318] [Other Feature Feedback, Questions, & Ideas]
Hi all, welcome to the latest check-in for the Profile category ✨

This time I’ve got some fresh releases that the team has been hard at work on over the last few months.

### Profile pronouns 
You can now [set your pronouns](https://github.blog/changelog/2023-03-01-add-pronouns-to-your-github-profile/) to display on your profile. These will show up on your main profile or on profile hover info cards. You can choose from a few default options or input your own custom pronouns.

### Mastodon social links
You can now [add Mastodon links](https://github.com/orgs/community/discussions/5720#discussioncomment-4847866) to your GitHub profile. Edit your profile and paste in a link to your Mastodon account in the “social profiles” section and GitHub will link to your profile with a cute little Mastodon icon. 🦣

As a bonus, if you link back to GitHub from Mastodon, GitHub will automatically verify that link on your Mastodon profile!

**Comment by AzizBenIsmail** [comment]
Okay

**Comment by 60355322264** [comment]
Tüm Tüm hesabıma bağlı işletme ve her şeyi silip ve devredışı bırakmamasını istiyorum

**Comment by pie972** [comment]
I love the 'set your pronouns' feature a lot. Thanks ^^

**Reply by otter-computer** [reply]
It's great, and has been a long time coming! So glad it finally released ☺️ ✨

**Comment by Souvik0901** [comment]
Yeah your profile is really good to see and it has some cool featured.Love it.

**Comment by oscar630** [comment]
Todavía no le entiendo nada a esta jaja pero bueno le iré aprendiendo si me podrán ayudar a confirmar unos préstamos personales que tengo y no los encuentro

**Reply by technerdfromhellv1** [reply]
You need help with loans? Like elaborate for me please?

**Comment by Calypso66** [comment]
Ok
